### DISCUSSION TITLE
目前有哪些值得推荐的插件

### DISCUSSION THREAD

**Discussion by LCehoennardo** [obsidianzh/forum#11] [Q&A常见问题]
目前有哪些值得推荐的插件

**Comment by Quorafind** [comment]
首先是 minimal setting & hidder
然后是 
-  Text {{expand}}
-  Templater
-  Day Planner
-  Calendar
-  Sliding Panes
-  WorkBench

**Reply by LCehoennardo** [reply]
minimal setting & hidder是什么，找不到这个插件，可以给个地址吗？

**Reply by makerjackie** [reply]
hidder-> 在plugin闪电中搜索 hider（隐藏部分obsidian的控件/UI）
minimal setting 同样可以搜索到，或者你看：https://github.com/kepano/obsidian-minimal-settings
![image](https://user-images.githubusercontent.com/18050469/101739738-ebfafd00-3b02-11eb-96b2-b48c1f07562c.png)

**Reply by LCehoennardo** [reply]
谢谢

**Reply by makerjackie** [reply]
ps: 我感觉 minimal 太简洁了，用不太来

**Comment by lxyntz** [comment]
https://getquicker.net/Share/Actions?exe=obsidian.exe

**Comment by makerjackie** [comment]
我也来推荐几个

* Mind Map：将文档转换为思维导图
![image](https://user-images.githubusercontent.com/18050469/101726026-b519ec80-3aec-11eb-9647-8ddb0e7c1451.png)


对于程序员用户，我推荐（我感觉这几个插件应当直接整合进正式版里头）
* Editor Syntax Highlight：代码块高亮
* Copy button for code blocks：预览模式中给代码块增加 Copy 按钮
* Obsidian Vimrc Support：可以配置vim啦，虽然仅支持部分功能（但解决了 复制到 系统剪贴板的痛点）

**Reply by appdev** [reply]
Mind Map 如果写了代码就不能显示了。

**Comment by windoversnow** [comment]
obsidian还没有专用剪藏插件吗？类似joplin那种，可以把网上图片剪藏到本地的？

**Reply by cirvoe** [reply]
[MaoXian web clipper - Chrome 网上应用店](https://chrome.google.com/webstore/detail/maoxian-web-clipper/kjahokgdcbohofgdidndeiaigkehdjdc)
这个插件可以下载图片到本地，但网络上有许多是webp格式，obsidian无法显示，而且会显示在图谱上，但webp的网络图片就可以显示。

[Webp support - Feature requests - Obsidian Forum](https://forum.obsidian.md/t/webp-support/805)

**Reply by LoveOctocat** [reply]
> obsidian还没有专用剪藏插件吗？类似joplin那种，可以把网上图片剪藏到本地的？

用剪悦插件的离线 Markdown 功能，图片直接保存为 base64（图片多可能会卡）

**Comment by windoversnow** [comment]
通用剪藏工具我以前也用 剪藏下来的markdown文件 图片链接还是指向网络的

其实可以借助外部编辑器 铁破啦 ，但铁破啦对微信公众号文章的图片不友好 不会图片重命名 保存下来的图片命名都是640，就重名了

![image](https://user-images.githubusercontent.com/75920913/102003631-7da57d00-3d44-11eb-86e3-2d03890e6b9b.png)

**Reply by windoversnow** [reply]
插入图片好像没显示出来，可以[看这里](https://s3.ax1x.com/2020/12/13/reMLaq.png)

**Reply by makerjackie** [reply]
铁破啦 是 typora 的意思吧，截图是思源笔记的？

**Reply by windoversnow** [reply]
> 铁破啦 是 typora 的意思吧，截图是思源笔记的？

是的 是的，😊

**Comment by grisbear** [comment]
https://github.com/Liamballin/ObsidianBookmark 剪藏工具

**Reply by ghost** [reply]
请问这个扩展怎么弄？
我现在是把它github的文件下载下来然后把extension文件放进chrome的拓展里，然后就不知道下一步了

**Reply by makerjackie** [reply]
这玩意挺麻烦的..我看了一下，打算等出obsidian社区插件再用了

**Reply by ghost** [reply]
> 这玩意挺麻烦的..我看了一下，打算等出obsidian社区插件再用了

原来还要下一个exe，不知道有没有剪藏工具能像remnote的拓展一样

**Reply by ghost** [reply]
下载exe后还是不会用。。。

**Reply by grisbear** [reply]
下载后，安装扩展。

打开 obsidian-bookmark.exe，
填写
- Vault folder 存放你要保存文档的位置
- Save location if different from vault root folder 这个没填
- Attatchments folder for image downloads 存放图片的位置

点击 Start Server，就可以运行了。

浏览器打开一篇文章，鼠标选取要保存的内容，然后点击扩展图标。

**Comment by fred1357944** [comment]
幫助家人更快進入obsidian 而錄製的plug-in 影片也跟大家分享。陸續更新中
[Workbench |Query |Slide |icons | iframe|Focus mode] x Obsidian
https://youtu.be/ccfdudNS544
[URL to markdown & Mind map]x Obsidian（中文）
https://youtu.be/RMWxbM6is7A
 [Obsidian] Get the note headings at once with "note refactor" and "fold heading and list"
https://youtu.be/38gqblPB-4U
 [Obsidian]Recent/Tag Wrangler/Collapse all/
Typewriter/Hotkey++/Markdown formatting assistant
https://youtu.be/Glnjn6Qgd1U
